plat/common/x86/cpu_native.c

```diff
@@ -55,6 +66,17 @@ void system_off(void)
 	outb(0x64, 0xFE);
 #endif /* CONFIG_KVM_VMM_FIRECRACKER */
 
+#ifdef CONFIG_KVM_VMM_QEMU
+	/* If we are crashing then try to exit QEMU with the isa-debug-exit
+	 * device.
+	 * Should be harmless if it is not present. This is used to enable
+	 * automated tests on virtio.  Note that the actual QEMU exit() status
+	 * will be 85 ('S', 42 << 1 | 1).
+	 */
+	if (request == UKPLAT_CRASH)
+		qemu_debug_exit(42);
```

Comment:
Would it make sense to have a macro for `42` - this doesn't tell me anything? Why `42`? What magic is this?